helm/templates/_helpers.tpl

```diff
@@ -0,0 +1,52 @@
+{{- define "notary.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{- define "notary.fullname" -}}
+{{- if .Values.fullnameOverride -}}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- $name := default .Chart.Name .Values.nameOverride -}}
+{{- if contains $name .Release.Name -}}
+{{- .Release.Name | trunc 63 | trimSuffix "-" -}}
+{{- else -}}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+{{- end -}}
+{{- end -}}
+
+{{- define "notary.serverdburl" -}}
+{{- if eq .Values.storage.flavor "mysql" -}}
+{{- if .Values.storage.enabled -}}
+{{ .Values.server.storageCredentials.user }}:%% .Env.PASSWORD %%@tcp(notary-db:3306)/notaryserver
```

Comment:
According to the documentation the connection string should include: `?parseTime=true`.

Eventhough if it's not present it would be added by default, it should be better if it is included in the chart.

There are 3 more ocurrences of the connection string, maybe it's a good idea to use a variable.